standard.tex

```diff
@@ -26,67 +26,76 @@
-\item If you're making a crimped connection, use a proper crimp tool if at all possible. Pliers work in a pinch, but the connection is messier and more likely to fail. Good crimp tools are expensive but worth it. (If you're in MME, you can borrow one from MESS.)
-\item If you absolutely need to solder stranded core to something, make sure it's very strain relieved (I personally like P clamps, but zip ties work too).
-\item Crimped connections should be done on stranded wire only. Don't crimp onto solid core. It's weak and it will break, often when you least want it to.
-\item All exposed wire should be heatshrunk; all heatshrink should be clear (IREC regulation).
-\item If you screw up a soldering job (we all do it), redo the job. When soldering surface mount components, inspect solder joints with a small magnifying glass.
+\item Crimped connections are preferable to soldered connections. Bad crimped connections are easier to identify than bad solder joints. Additionally, it is easier to fix a bad crimp in the field since it does not require powered tooling.
+\item If you're making a crimped connection, use a proper crimp tool if at all possible. Pliers are not an acceptable alternative (see ESRA safety critical wiring guidelines). We have proper crimp tools for: generic butt splice and ring/spade connectors, Dupont connectors, and RJ45 connectors.
+\item Use ferrules for stranded wire that goes into a screw terminal.
+\item Crimped connections should be done on stranded wire only.
+\item Wire should only be soldered to connectors that are specifically made for it (eg. connectors with solder cups). Stranded wire should be used in the vast majority of cases since solid core wire becomes brittle and prone to breakage (exception: wire soldered onto protoboard should be solid core).
+\item All solder joints should be strain relieved with zipties or P clamps.
+\item If you screw up a soldering job, redo the job. When soldering surface mount components, inspect solder joints with a small magnifying glass.
 \item All connections must be tug proof.
+\item All exposed wire should be heatshrunk; all heatshrink should be clear (IREC regulation).
 \item If a connector could be asymmetric, it should be asymmetric, since asymmetric connectors are much harder to plug in backwards.
+\item Use positive fit connectors wherever possible.
 \end{itemize}
-%end of connections subsection -- }}}
+%-- }}}
 
-%start of wiring and assembly subsection -- {{{
+%Wiring and Assembly -- {{{
 \subsection{General Wiring \& Assembly}
 \begin{itemize}
+\item Any electrical system that may be unsafe to approach while powered should have a manual kill switch installed that:
+    \begin{itemize}
+    \item disables all energetics or potentially unsafe components.
+    \item can be operated without requiring any disassembly.
+    \end{itemize}
```

Comment:
not just operated without dissassembly - without taking rocket off of tower

Reply:
Ideally, yes. I do not know if that's feasible when the rocket is tall enough that you can't reach most of it. Anything remote defeats the purpose.

Reply:
Even magswitches aren’t terrible. Bringing the rocket down with armed deployment charges is something we used to mock other teams about, we should have some hard rules against it